locales/base/translation.json

```diff
@@ -1348,6 +1348,7 @@
     "transactionDetails": "Transaction Details",
     "paymentMethodVia": "Connected via {{providerName}}",
     "receiveAmount": "You will receive",
+    "feeDisclaimer": "Your bank may charge additional fees for your transfer.",
```

Comment:
This string is bank specific. Shouldn't it support mobile money as well?

Reply:
Good catch!